snippet-extractor-output/snippets.yaml

```
@@ -13357,6 +13357,50 @@
         console.log("Search term: " + result.searchTerm + " => Count: " + length);
       });
     });
+'Word.Body#type:member':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/50-document/manage-footnotes.yaml
+
+    // Get the referenced note's type, which is footnote.
+
+    await Word.run(async (context) => {
+      const footnotes = context.document.body.footnotes;
+      footnotes.load("items");
+      await context.sync();
+
+      const referenceNumber = $("#input-reference").val();
+      const mark = (referenceNumber as number) - 1;
+      const item = footnotes.items[mark];
+      console.log(`Note type of footnote ${referenceNumber}: ${item.type}`);
+
+      item.body.load("type");
+      await context.sync();
+
+      console.log(`Body type of note: ${item.body.type}`);
+    });
+'Word.BodyType:enum':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/50-document/manage-footnotes.yaml
+
+    // Get the referenced note's type, which is footnote.
```

Perhaps this as an explanation: 
```suggestion
    // Get the referenced note's item type and body type, which are both footnote.
```